Sources/GRPCHTTP2TransportNIOPosix/HTTP2ServerTransport+Posix.swift

```diff
@@ -174,6 +178,15 @@ extension HTTP2ServerTransport {
         }
       }
 
+      #if canImport(NIOSSL)
+      let nioSSLContext: NIOSSLContext?
+      if let tlsConfig = self.config.transportSecurity.tlsConfig {
+        nioSSLContext = try NIOSSLContext(configuration: TLSConfiguration(tlsConfig))
```

Comment:
Can we wrap this up in our own error and have this as the `cause`? That way we can make it clear that the tis config was invalid (not sure how clear NIOSSLErrors are, especially if they come from BoringSSL)